.rubocop.yml

```diff
@@ -0,0 +1,53 @@
+# use the shared Yast defaults
+inherit_from:
+  /usr/share/YaST2/data/devtools/data/rubocop_yast_style.yml
+
+# TODO: this need some non-trivial refactoring...
+Metrics/AbcSize:
+  Max: 111
+
+# TODO: this need some non-trivial refactoring...
+Metrics/BlockNesting:
+  Max: 5
+
+# TODO: this need some non-trivial refactoring...
+Metrics/ClassLength:
+  Max: 450
+
+# TODO: this need some non-trivial refactoring...
+Metrics/CyclomaticComplexity:
+  Max: 25
+
+Metrics/LineLength:
+  Max: 100
+
+# TODO: this need some non-trivial refactoring...
+Metrics/MethodLength:
+  Max: 96
+
+# TODO: this need some non-trivial refactoring...
+Metrics/PerceivedComplexity:
+  Max: 25
+
+# FIXME: It seems the cop does not handle this complex case correctly,
+# disable it for now on that file.
+Lint/ShadowedException:
+  Exclude:
+    - 'src/lib/registration/connect_helpers.rb'
```

Comment:
In general whole file should be cleaned up, I guess many rules are not needed. Just remove everything except the `inherit_from` option, then run Rubocop and see what fails. Then either fix that or disable.